README.md

```diff
@@ -101,6 +101,11 @@ Example command:
 - GitHub: [GitHub](https://github.com/Jaymelfah)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/jeremiah-ekow-melfah-a4402a161/)
 
+👤 **Olivier Kango**
+
+- GitHub: [GitHub](https://github.com/Olivier-Kango)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/olivier-kango-b990601b8)
+
 
```

Comment:
- [OPTIONAL] It will be nice to update your getting started section with the command to run your test locally as this will be of help to non technical users